pkgs/development/libraries/c-ares/default.nix

```diff
@@ -7,11 +7,12 @@
 
 let self =
 stdenv.mkDerivation rec {
-  name = "c-ares-1.15.0";
+  pname = "c-ares";
+  version = "1.17.1";
 
   src = fetchurl {
-    url = "https://c-ares.haxx.se/download/${name}.tar.gz";
-    sha256 = "0lk8knip4xk6qzksdkn7085mmgm4ixfczdyyjw656c193y3rgnvc";
+    url = "https://c-ares.haxx.se/download/${pname}-${version}.tar.gz";
```

Comment:
you know, maybe we'd have fewer cves if we didn't fetch source code from a haxxer domain

Reply:
Ugh, it gets worse! https://daniel.haxx.se/blog/2016/01/19/subject-urgent-warning/